src/Akka.CQRS.Hosting.Tests/TradeProcessor.cs

```diff
@@ -2,259 +2,275 @@
 using Akka.Hosting;
-using Akka.Persistence.Sql;
 using Akka.CQRS.Infrastructure;
 using Akka.CQRS.TradeProcessor.Actors;
-using Petabridge.Cmd.Host;
-using Petabridge.Cmd.Cluster;
-using Petabridge.Cmd.Cluster.Sharding;
-using Petabridge.Cmd.Remote;
-using Microsoft.Extensions.DependencyInjection;
 using Akka.Util;
-using Akka.Bootstrap.Docker;
-using Microsoft.Extensions.Options;
+using Akka.Persistence.SqlServer.Hosting;
+using Akka.Remote.Hosting;
+using System.Net;
+using FluentAssertions;
+using Akka.Cluster.Sharding;
 using Akka.Cluster.Tools.Singleton;
 using Akka.CQRS.Pricing.Actors;
-using Akka.CQRS.Pricing.Cli;
 using Akka.Persistence.Query;
-using Akka.Persistence.Sql.Query;
-using Testcontainers.MsSql;
+using Akka.Persistence.Query.Sql;
+using Microsoft.Win32;
+using static LinqToDB.Reflection.Methods;
 
 namespace Akka.CQRS.Hosting.Tests
 {
-    public class TradeProcessor : IAsyncLifetime
+    public class TradeProcessor : Akka.Hosting.TestKit.TestKit
     {
-        private readonly string _sqlConnectionString = "Server=sql,1433;User Id=sa;Password=This!IsOpenSource1;TrustServerCertificate=true";
+        private readonly string _sqlConnectionString = "Server=sql,1633;User Id=sa;Password=This!IsOpenSource1;TrustServerCertificate=true";
         private readonly string _sqlProviderName = "SqlServer.2019";
+        private string _configTradeProcessor = ConfigurationFactory.ParseString(@"
+            akka {{
+	actor {{
+		provider = cluster
+	}}
+						
+	remote {{
+		dot-netty.tcp {{
+            hostname = ""127.0.0.1""
+            port = 5055
+        }}
+	}}			
 
-        private MsSqlContainer _dockerContainer = null!;
-        private IContainer _lighthouse = null!;
+	cluster {{
+		#will inject this node as a self-seed node at run-time
+		seed-nodes = [""akka.tcp://AkkaTrader@127.0.0.1:5055""] 
+		roles = [""trade-processor"" , ""trade-events""]
 
-        public TradeProcessor(ITestOutputHelper output)
-        {
-            Output = output;
-        }
+		pub-sub{{
+			role = ""trade-events""
+		}}
 
-        public ITestOutputHelper Output { get; }
-        public async Task DisposeAsync()
-        {
-            await _dockerContainer.StopAsync();
-            await _lighthouse.StopAsync();
-        }
+		sharding{{
+			role = ""trade-processor""
+		}}
+	}}
 
-        public async Task InitializeAsync()
-        {
-            #region SQL
-            _dockerContainer = new MsSqlBuilder()
-                .WithName("sql")
-                .WithImage("mcr.microsoft.com/mssql/server:2019-latest")
-                .WithPortBinding(1633, 1433)
-                .WithPassword("This!IsOpenSource1")
-                .WithEnvironment(new Dictionary<string, string>()
-                {
-                        { "ACCEPT_EULA", "Y" }
-                })
-                .WithCleanUp(true)
-                .Build();
+	persistence{{
+		journal {{
+		    plugin = ""akka.persistence.journal.sql""

```

Comment:
Use `Akka.Persistence.SqlServer.Hosting` and use the `.WithSqlServerPersistence()` extension method instead.